tardis/analysis.py

```diff
@@ -2,12 +2,15 @@
 
 import re
 import os
+import logging
```

Comment:
Why are there changes in this file? They don't do anything.

Reply:
I wanted to ask about this. Do we need to log exceptions for this part of the source code 🤔